common/network-types/Cargo.toml

```diff
@@ -69,5 +69,5 @@ hopr-async-runtime = { workspace = true, features = ["runtime-tokio"] }
 [package.metadata.cargo-machete]
 ignored = ["serde_bytes"]
 
-[package.metadata.cargo-shear]
-ignored = ["serde_bytes"]
+# [package.metadata.cargo-shear]
```

Comment:
Why not delete right away if not needed?

Reply:
Remnant, there's a mismatch between the version I have on the system and the one used by the devShell, fixed.